assets/src/components/detoursTable.tsx

```diff
@@ -65,12 +69,15 @@ export const DetoursTable = ({
 
 const PopulatedDetourRows = ({
   data,
+  status,
   onOpenDetour,
 }: {
   data: SimpleDetour[]
+  status: DetourStatus
   onOpenDetour: (detourId: number) => void
 }) => {
-  const epochNowInSeconds = useCurrentTimeSeconds()
+  const epochNow = useCurrentTime()
+  const epochNowInSeconds = epochNow.valueOf() / 1000
```

Comment:
non-blocking: what's the purpose of this change? Looks like `useCurrentTime` uses the same underlying logic as `useCurrentTimeSeconds` (other than the divide by seconds)